src/main/java/com/matt/forgehax/mods/AutoTotemMod.java

```diff
@@ -21,7 +22,17 @@
   public AutoTotemMod() {
     super(Category.COMBAT, "AutoTotem", false, "Automatically move totems to off-hand");
   }
-  
+
+  private final Setting<Boolean> MenuPatch =
+      getCommandStub()
+          .builders()
+          .<Boolean>newSettingBuilder()
+          .name("MenuPatch")
```

Comment:
you should use camel case or underscores for field names, but the `.name` should not be capitalized and underscores/whitespace should be hyphed (ex: `.name("allow-gui")`)